java/lang/path_traversal.yml

```
@@ -61,15 +61,33 @@ languages:
 trigger:
   match_on: presence
 metadata:
-  description: SQL injection threat detected
+  description: Possible path traversal vulnerability detected
   remediation_message: |
-    classes used:
-    # java.io.File
-    # java.io.FileOutputStream
-    # java.io.FileInputStream
-    # org.apache.commons.io.FilenameUtils.getName
+    ## Description
+
+    Allowing unsanitized user input in path resolution methods means an attacker could gain access to files and folders outside of the intended scope.
+
+    ## Remediations
+    ❌ Avoid wherever possible
+
+    ✅ Sanitize user input when resolving paths, for example:
+    - Use `FilenameUtils.getName()` to mitigate against unwanted patterns in the path (such as `\..\..`)
```

```suggestion
    Use `FilenameUtils.getName()` to mitigate against unwanted patterns in the path (such as `\..\..`)
```
Probably shouldn't be a list item here.